### PR TITLE
Pricing page iteration

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

Iterated on the pricing page according to the related issues.

## Related Issue(s)

[#1814](https://github.com/FlowFuse/website/issues/1814) 
https://github.com/FlowFuse/website/issues/1959

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/_includes/feature_lists/features-table-base.njk [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <li class="ff-feature--column-header">
             <span class="sticky left-0 h-full"></span>
             {% if sections[0].hosting == "cloud" %}
-            <label class="mr-2">
+            <label class="mr-2 ">
```
Comment (ZJvandeWeg) [inline src/_includes/feature_lists/features-table-base.njk:6]: ✂️

File: src/_includes/feature_lists/cloud.njk [hunk 2]
```diff
@@ -96,56 +119,16 @@
...
             values: [null, null, 'check'],
             info: "<p>FlowFuse can be configured to run with your own Single Sign-On (SSO) provider. Allow users to access FlowFuse with a single set of login credentials, improving security, user experience, and save them time.</p>"
-        }, {
-            id: "ha",
-            label: "High Availibility",
-            values: [null, null, 'check'],
-            info: "<p>Leverage horizontal scaling for reliable and scalible processing of your data through Node-RED.</p>"
-        }]
-    }, {
-        label: "Support & SLA",
```
Comment (ZJvandeWeg) [inline src/_includes/feature_lists/cloud.njk:126]: @zackwasli WDYT about removing support from the pricing page?

File: src/_includes/feature_lists/features-table-base.njk [hunk 3]
```diff
@@ -63,6 +63,7 @@
             {% endif %}
             {% endfor %}
         </li>
+    </ul>
```
Comment (ZJvandeWeg) [inline src/_includes/feature_lists/features-table-base.njk:66]: 💪

File: src/_includes/feature_lists/self-hosted.njk [hunk 4]
```diff
@@ -1,6 +1,35 @@
 {%
     set sections = [{
-        label: "Features",
+        label: "No-Ops Node-RED",
```
Comment (ZJvandeWeg) [inline src/_includes/feature_lists/self-hosted.njk:3]: At what stage will we unify the too feature_list files? :)

File: src/_includes/feature_lists/self-hosted.njk [hunk 5]
```diff
@@ -13,41 +42,55 @@
...
+        }]
+    }, {
+        label: "Security & Compliance",
+        hosting: "self-hosted",
+        rows: [{
             id: "nr-hosting",
             label: "Node-RED Hosting",
-            values: ['check', 'check'],
+            values: [null, null],
             info: "<p>FlowFuse hosts Node-RED instances.</p>"
```
Comment (ZJvandeWeg) [inline src/_includes/feature_lists/self-hosted.njk:82]: It still does, why this change?

File: src/pricing/index.njk [hunk 6]
```diff
@@ -5,10 +5,20 @@ sitemapPriority: 0.90
 meta:
     description: Discover the pricing options for FlowFuse, featuring Starter, Team, and Enterprise plans. Benefit from volume discounts on Team and Enterprise plans. Get started with a 14-day trial period for the Starter plan.
     keywords: FlowFuse pricing, Cost, Plans, Features, Cloud, Self-hosted
+    faq:
```
Comment (ZJvandeWeg) [inline src/pricing/index.njk:8]: @zackwasli Can you update this list to include the true most frequent questions? E.g. why pricing for self-hosted is the same as Cloud

File: src/pricing/index.njk [hunk 7]
```diff
@@ -273,54 +330,80 @@ hubspot:
...
+                    gives you the chance to explore all the features and functionalities of FlowFuse in your own
+                    environment. To start your trial, just fill out the form.
                 </p>
             </div>
             {% include hubspot.script %}
         </div>
     </div>
 </div>
 
+<!-- Frequently Asked Questions -->
```
Comment (ZJvandeWeg) [inline src/pricing/index.njk:379]: Isn't there a partial for FAQs? If not, we should introduce that in this PR

File: src/_includes/feature_lists/cloud.njk (hunk 2 above)
Comment (Yndira-E) [reply src/_includes/feature_lists/cloud.njk:126]: Entirely? The plan was to remove it here and add it to the hero section, and I've done just that. How crucial is this aspect for potential customers when they evaluate which plan best suits their needs? If you say it isn't, I think we can remove it.

Comment (zackwasli) [reply src/_includes/feature_lists/cloud.njk:126]: If we're talking about this section, I think it displays useful info around support (specifically SLAs). What would be the reasoning to remove it?

Screenshot 2024-04-26 at 10 07 08 AM

File: src/pricing/index.njk (hunk 6 above)
Comment (zackwasli) [reply src/pricing/index.njk:8]: Yes here is my answers from the other day: https://github.com/FlowFuse/website/issues/1959#issuecomment-2073175913

Does FlowFuse offer annual discounts?
-Yes, we offer a 15% discount for annual subscriptions.

Is there a minimum for purchase?
-Yes, 5 instances on Team and 10 instances on Enterprise.

Why isn't there an option for self-hosted Team?
-Due to the configuration and support required for a self-hosted deployment, it is only possible through the Enterprise tier at this time.

Why is self-hosted the same price as cloud?
-Self-hosted environments require more support and configuration than FlowFuse Cloud, which offset any resources saved.

File: src/_includes/feature_lists/self-hosted.njk (hunk 5 above)
Comment (Yndira-E) [reply src/_includes/feature_lists/self-hosted.njk]: That wasn't right, I changed it back

File: src/pricing/index.njk (hunk 6 above)
Comment (Yndira-E) [reply src/pricing/index.njk:10]: I updated the list

File: src/pricing/index.njk (hunk 7 above)
Comment (Yndira-E) [reply src/pricing/index.njk:384]: There is now! 😉

File: src/_includes/feature_lists/cloud.njk (hunk 2 above)
Comment (Yndira-E) [reply src/_includes/feature_lists/cloud.njk:126]: The plan was to remove it here and to add only the SLA info in the hero section, and I've done just that.

File: src/_includes/feature_lists/self-hosted.njk (hunk 4 above)
Comment (Yndira-E) [reply src/_includes/feature_lists/self-hosted.njk:3]: I wish I could say at this stage, but I spent some time trying to do it, and it wasn't working. I can give it another try some other time. I'd like to move on to other items on my to-do list if that's ok 🙂

File: src/_includes/feature_lists/self-hosted.njk [hunk 8]
```diff
@@ -13,11 +42,40 @@
...
+            label: "Access to Blueprints",
+            values: [null, 'check'],
+            info: "<p><a href=\"https://flowfuse.com/blog/2023/10/blueprints/\" target=\"blank\">FlowFuse's Blueprints</a> simplify Node-RED deployments by offering pre-built, customizable flows for specific use cases.</p>"
+        }]
+    }, {
+        label: "Security & Compliance",
+        hosting: "self-hosted",
+        rows: [{
             id: "nr-hosting",
             label: "Node-RED Hosting",
```
Comment (ZJvandeWeg) [inline src/_includes/feature_lists/self-hosted.njk:80]: Either we should remove this all together, or put it the No-Ops section.

File: src/pricing/index.njk [hunk 9]
```diff
@@ -2,13 +2,28 @@
 align: center
 layout: layouts/page.njk
 sitemapPriority: 0.90
+faqTitle: "Frequently Asked <span class='text-indigo-600'>Questions</span>"
+faqSubtitle: "If you have any questions not addressed here, don't hesitate to contact us through our <a href='/contact-us'>Contact Us</a> page"
 meta:
     description: Discover the pricing options for FlowFuse, featuring Starter, Team, and Enterprise plans. Benefit from volume discounts on Team and Enterprise plans. Get started with a 14-day trial period for the Starter plan.
     keywords: FlowFuse pricing, Cost, Plans, Features, Cloud, Self-hosted
+    faq:
+        - question: "Does FlowFuse offer annual discounts?"
```
Comment (ZJvandeWeg) [inline src/pricing/index.njk:11]:
```suggestion
        - question: "Does FlowFuse offer annual discounts for FlowFuse Cloud?"
```

File: src/pricing/index.njk [hunk 10]
```diff
@@ -2,13 +2,28 @@
...
+          answer: >
+            Yes, we offer a 15% discount for annual subscriptions.
```
Comment (ZJvandeWeg) [inline src/pricing/index.njk:13]:
```suggestion
            Yes, we offer a 15% discount for annual subscriptions. Please <a href="/contact-us/">reach out</a> to request the discount.
```

File: src/pricing/index.njk [hunk 11]
```diff
@@ -2,13 +2,28 @@
...
+        - question: "Is there a minimum for purchase?"
+          answer: >
+            Yes, 5 instances on Team and 10 instances on Enterprise.
+        - question: "Why isn't there an option for self-hosted Team?"
+          answer: >
+            Due to the configuration and support required for a self-hosted deployment, it is only possible through the Enterprise tier at this time.
+        - question: "Why is self-hosted the same price as cloud?"
+          answer: >
+            Self-hosted environments require more support and configuration than FlowFuse Cloud, which offset any resources saved.
```
Comment (ZJvandeWeg) [inline src/pricing/index.njk:22]:
```suggestion
        - question: "Why is self-hosted the same price as cloud?"
          answer: >
            Self-hosted environments require more support and configuration than FlowFuse Cloud, which offset any resources saved.
        - question: "Are flat-fee or site-wide licenses available?"
          answer: >
            To allow business to standardize on FlowFuse and Node-RED for all their data acquisition, visualization, and interaction it's possible to purchase a flat-fee license to deploy an unlimited amount of Node-RED instances through FlowFuse.
```

**Comment by ZJvandeWeg** [comment]
@Yndira-E One open question in the review, and another one here: Are we set up correctly to measure the performance of the pricing page after merging?

File: src/_includes/feature_lists/self-hosted.njk (hunk 8 above)
Comment (Yndira-E) [reply src/_includes/feature_lists/self-hosted.njk]: I've moved it under No-Ops

**Comment by Yndira-E** [comment]
@ZJvandeWeg 

> Are we set up correctly to measure the performance of the pricing page after merging?

I've already added/updated CTAs to be captured by PostHog, I think it's ready to set up a new dashboard.

And I've just pushed a change adding the form on the dedicated section, so it will be easier to measure where those leads are coming from in HubSpot.
